src/Contrib/OtlpGrpc/Exporter.php

```diff
@@ -48,24 +48,39 @@ public function __construct(
         int $timeout = 10,
         TraceServiceClient $client = null
     ) {
-        // Set default values based on presence of env variable
-        $this->insecure = $this->getBooleanFromEnvironment(Env::OTEL_EXPORTER_OTLP_INSECURE, $insecure);
-        if (!empty($certificateFile) || EnvResolver::hasVariable(Env::OTEL_EXPORTER_OTLP_CERTIFICATE)) {
-            $this->certificateFile = $this->getStringFromEnvironment(Env::OTEL_EXPORTER_OTLP_CERTIFICATE, $certificateFile);
-        }
-        $this->compression = $this->getEnumFromEnvironment(
-            Env::OTEL_EXPORTER_OTLP_COMPRESSION,
-            $compression ? Values::VALUE_GZIP : Values::VALUE_NONE
-        );
-        $this->timeout = $this->getIntFromEnvironment(Env::OTEL_EXPORTER_OTLP_TIMEOUT, $timeout);
+
+        $this->insecure = $this->hasEnvironmentVariable(Env::OTEL_EXPORTER_OTLP_INSECURE) ?
+            $this->getBooleanFromEnvironment(Env::OTEL_EXPORTER_OTLP_INSECURE, $insecure) :
+            $this->getBooleanFromEnvironment(Env::OTEL_EXPORTER_OTLP_INSECURE, $insecure);
+
+        if (!empty($certificateFile) || EnvResolver::hasVariable(Env::OTEL_EXPORTER_OTLP_CERTIFICATE)) 
```

Comment:
Could you use `hasEnvironmentVariable`here as well?

Reply:
Yes, not sure how I missed that one. Will make the changes